src/deck/drivers/src/ledring12.c

```diff
@@ -42,12 +42,13 @@
 #include "log.h"
 #include "pulse_processor.h"
 #include "mem.h"
+#include "autoconf.h"
 
 #define DEBUG_MODULE "LED"
 #include "debug.h"
 
-#ifdef LED_RING_NBR_LEDS
-#define NBR_LEDS  LED_RING_NBR_LEDS
+#ifdef CONFIG_DECK_LEDRING_NBR_LEDS
```

Comment:
My understanding is that since `DECK_LEDRING_NBR_LEDS` is an int, kconfig should already have declared it with the proper default value. So the `#ifdef`s should not be required anymore.

Reply:
Fixed in 0afe821